cclib/method/ddec.py

```diff
@@ -274,6 +274,38 @@ def calculate(self, indices=None, fupdate=0.05):
         self.logger.info("Conditioning charge densities.")
         self.condition_densities()
 
+        # STEP 4-7
+        # Optimize weights on each grid point
+        kappa = [0.0 for x in self.data.atomnos]
+        update_kappa = False
+        self.N_A = []
+        steps = 4
+        while steps < 5:  # change this to 8 after implementing kappa routine
+            self.logger.info("Optimizing charge weights (Step {}).".format(steps))
+            self.N_A.append(self.calculate_w_and_u())
+
+            # Update kappa as described in (4) of doi: 10.1039/c6ra04656h
```

Comment:
(4) doesn't look like the right reference.

Reply:
I've updated this to a correct reference. Thanks!